TransferMethod/Tests/Responses/BankAccountIndividualUpdateResponse.json

```diff
@@ -0,0 +1,7 @@
+{
+  "userToken": "usr-0001",
+  "token": "trm-0001",
+  "status": "ACTIVATED",
+  "createdOn": "2020-12-11T13:34:35.762Z"
+}
+
```

Comment:
response will always contain all the fields, refer response section in https://docs.hyperwallet.com/content/api/v3/resources/users/update

Reply:
> https://docs.hyperwallet.com/content/api/v3/resources/users/update

I see the mentioned URL is for Update User.
I got this response for updating Bank Account (https://docs.hyperwallet.com/content/api/v3/resources/bank-accounts/update?profileType=Individual&accountType=BANK_ACCOUNT&country=United+States&currency=USD)
Please do confirm on this. Thank You